src/main/java/core/basesyntax/OperationHandler.java

```diff
@@ -0,0 +1,7 @@
+package core.basesyntax;
+
+public interface OperationHandler {
+    void setFruitTransfer(FruitTransfer fruitTransfer);
```

Comment:
This looks like an ordinal setter without any additional logic, so let's simplify your code
```suggestion
```